app/frontend/src/features/communities/leaveReference/formSteps/Rating.tsx

```diff
@@ -0,0 +1,86 @@
+import { Typography, useMediaQuery, useTheme } from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import RatingsSlider from "components/RatingsSlider/RatingsSlider";
+import TextBody from "components/TextBody";
+import {
+  NEXT,
+  PRIVATE_ANSWER,
+  QUESTION_MARK,
+  RATING_EXPLANATION,
+  RATING_QUESTION,
+  REFERENCE_FORM_HEADING,
+} from "features/communities/constants";
+import {
+  ReferenceContextFormData,
+  useReferenceData,
+} from "features/communities/leaveReference/ReferenceDataContext";
+import {
+  ReferenceFormProps,
+  useReferenceStyles,
+} from "features/communities/leaveReference/ReferenceForm";
+import React from "react";
+import { Controller, useForm } from "react-hook-form";
+import { useHistory, useParams } from "react-router-dom";
+import { leaveReferenceBaseRoute } from "routes";
+
+export default function Rating({ user }: ReferenceFormProps) {
+  const history = useHistory();
+  const classes = useReferenceStyles();
+  const theme = useTheme();
+  const isMdOrWider = useMediaQuery(theme.breakpoints.up("md"));
+  const { referenceType, hostRequest } = useParams<{
+    referenceType: string;
+    hostRequest?: string;
+  }>();
+  const { data, setValues } = useReferenceData()!;
+  const { control, handleSubmit, errors } = useForm<ReferenceContextFormData>({
+    defaultValues: {
+      rating: data.rating,
+    },
+  });
+
+  const onSubmit = (values: ReferenceContextFormData) => {
+    setValues(values);
+    hostRequest
+      ? history.push(
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/${hostRequest}/reference`
+        )
+      : history.push(
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/reference`
+        );
+  };
+
+  return (
+    <form className={classes.form} onSubmit={handleSubmit(onSubmit)}>
+      <Typography variant="h1">
```

Comment:
```suggestion
      <Typography variant="h2">
```
I think the user name on the left of the profile is already an h1 and it's good practice to **not** have more than one level one heading for good accessibility